clients/src/main/resources/common/message/FindCoordinatorRequest.json

```diff
@@ -18,6 +18,8 @@
   "type": "request",
   "listeners": ["zkBroker", "broker"],
   "name": "FindCoordinatorRequest",
+  // Versions 0 was removed in Apache Kafka 4.0, Version 1 is the new baseline.
+  //
```

Comment:
KIP states

> FindCoordinator: none due to clients (Sarama, kafka-python) support – V0-V1 per the baseline

Does this mean we should not be removing any versions?

Reply:
Yes, good catch. It looks like this is a second case where we deprecated something incorrectly. I checked the KIP and this was indeed an update:

https://cwiki.apache.org/confluence/pages/diffpagesbyversion.action?pageId=235838251&selectedPageVersions=18&selectedPageVersions=17